gffquant/annotation/genecount_annotator.py

```
@@ -0,0 +1,49 @@
+import logging 
+
+import numpy as np
```

Ruff (0.8.2)

`numpy` imported but unused

Remove unused import: `numpy`

(F401)